beanvalidation/hibernate-validator/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>cloud.piranha.beanvalidation</groupId>
+        <artifactId>project</artifactId>
+        <version>20.6.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>piranha-beanvalidation-hibernatevalidator</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Piranha Bean Validation - Hibernate Validator integration</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>jakarta.validation</groupId>
+            <artifactId>jakarta.validation-api</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator-cdi</artifactId>
```

Comment:
Does this implementation require CDI? If so, is it pulling in transitive dependencies? If that is the case please exclude the CDI dependency here and document that it was excluded as the user should supply their own.

Reply:
Both BVal and Hibernate Validator requires CDI to be present, but none of them includes transitivly the CDI. I'll document that CDI must be present.